### PR TITLE
WIP: Repeated observations

### PR DESCRIPTION
**Pull request by mschauer**

### CODE REVIEW

File: scripts/simulate_repeated_part_obs.jl [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+FILENAME_OUT = joinpath(OUT_DIR,
+                     "test_path_part_obs_"*String(param)*".csv")
+
+K = 200
+
+P = FitzhughDiffusion(param, 10.0, -8.0, 15.0, 0.0, 3.0)
+# starting point under :regular parametrisation
+x0_ = ℝ{2}(-0.5, 0.6)
+# translate to conjugate parametrisation
+x0 = [regularToConjug(x0_, P.ϵ, 0.0) for k in 1:K]
```
Comment (mschauer) [inline scripts/simulate_repeated_part_obs.jl]: it is computed hiere

Comment (mmider) [reply scripts/simulate_repeated_part_obs.jl]: ah, missed it, sorry

File: src/repeated.jl [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    end
+    Σ = inv(𝓦 + inv(Matrix(prior.Σ)))
+    Σ = (Σ + Σ')/2 # eliminates numerical inconsistencies
+    μₚₒₛₜ = Σ * (μ + Vector(prior.Σ\prior.μ))
+    rand(Gaussian(μₚₒₛₜ, Σ))
+end
+
+
+# no blocking
+function updateParam!(::ObsScheme, ::ConjugateUpdt, 𝔅,
```
Comment (mschauer) [inline src/repeated.jl]: I just provided a special version of updateParam!

File: scripts/repeated.jl [hunk 3]
```diff
@@ -122,7 +125,7 @@ start = time()
                          P̃, Ls, Σs, numSteps, tKernel, priors, τ;
                          fpt=fpt,
                          ρ=0.975,
-                         dt=1/100,
+                         dt=1/1000,
```
Comment (mschauer) [inline scripts/repeated.jl]: Yes!

**Pull request merged**